alg/gdalwarper.cpp

```diff
@@ -1167,6 +1167,17 @@ CPLErr GDALWarpDstAlphaMasker(void *pMaskFuncArg, int nBandCount,
  * will be selected, not just those whose center point falls within the
  * polygon.</li>
  *
+ * <li>XSCALE: Ratio expressing the resampling factor along the target
+ * horizontal axis. Equals to one for no resampling, below one for downsampling
+ * and above one for upsampling. This is automatically computed, for each
+ * processing chunk, and may thus vary among them, depending on the
+ * shape of output regions vs input regions. Such variations can be undesired
+ * in some situations. If the resampling factor can be considered as constant
+ * over the warped area, setting a constant value can lead to more reproducible
+ * pixel output.</li>
```

Comment:
Describe `XSCALE=FROM_GRID_SAMPLING` ?

Reply:
Let's keep it as undocumented for now / implementation detail... (partly because I'm never satisfied with those settings, and also because this is tricky to describe accurately and in a understandable way)